tests/settings/e2e/steps/stakepool-settings.js

```diff
@@ -0,0 +1,83 @@
+import { When, Then } from 'cucumber';
+import en from '../../../../source/renderer/app/i18n/locales/en-US.json';
+
+const CUSTOM_SERVER_TEXT = en['settings.stakePools.smash.select.customServer'];
+const INVALID_SMASH_SERVER_TEXT = en['api.errors.invalidSmashServer'];
+const OFF_CHAIN_METADATA_SERVER_TEXT = en['settings.stakePools.smash.select.label'];
+const SMASH_SERVER_URL_TEXT = en['settings.stakePools.smashUrl.input.label'];
+const STAKE_POOL_TEXT = en['settings.menu.stakePools.link.label'];
+const YOUR_CHANGES_HAVE_BEEN_SAVED_SUCCESS_MESSAGE = en['inline.editing.input.changesSaved'];
+
+const CUSTOM_SERVER_DROPDOWN_OPTION = `//*[@class="SimpleInput_customValueWrapper"]//*[@value="${CUSTOM_SERVER_TEXT}"]`;
+const DAEDALUS_TOP_BAR_LOGO = '//*[@class="TopBar_topBar TopBar_withoutWallet"]';
+const NOT_A_VALID_SMASH_SERVER_ERROR_MESSAGE_ACTIVATED = `//*[@class="InlineEditingInput_errorMessage" and text()="${INVALID_SMASH_SERVER_TEXT}"]`;
+const OFF_CHAIN_METADATA_SERVER_SMASH_LABEL = `//*[contains(text(), "${OFF_CHAIN_METADATA_SERVER_TEXT}")]`;
+const SERVER_URL = "https://smash.cardano-testnet.iohkdev.io/";
+const SMASH_SERVER_URL_INPUT_BOX = `//*[@label="${SMASH_SERVER_URL_TEXT}"]`;
+const STAKE_POOL_CUSTOM_SERVER_INPUT_BOX_SUBMIT_BUTTON = '//*[@class="InlineEditingInput_button InlineEditingInput_okButton SimpleButton_root ButtonOverrides_root"]';
+const STAKE_POOL_CUSTOM_SERVER_INPUT_BOX_X_BUTTON = '//*[@class="InlineEditingInput_button InlineEditingInput_cancelButton SimpleButton_root ButtonOverrides_root"]';
+const STAKE_POOL_SERVER_DROPDOWN = '//*[@class="SimpleFormField_inputWrapper"]';
+const STAKE_POOL_SERVER_DROPDOWN_CUSTOM_OPTION = `//*[@class="ScrollbarsCustom-Content"]//span[text()="${CUSTOM_SERVER_TEXT}"]`;
+const STAKE_POOLS_SUBMENU_SETTINGS = `//*[@class="SettingsMenu_component"]//button[text()="${STAKE_POOL_TEXT}"]`;
+
+When(/^custom server is the default option$/, function() {
```

Comment:
Hi @daniloprates. Thank you for the review. On my side in selfnode all i can see are options for custom and "None fetch the data directly [screenshot](https://drive.google.com/file/d/1WQMUqzHNyvCyLA6BYDbwecfv-j2RW8M3/view?usp=sharing)  I had presumed that it was the same for everybody. What environment variables do i need to update ?

Reply:
Where are these environment variables set ? @daniloprates

Reply:
Manus the variable can be found at `daedalus.stores.staking.smashServerUrl`.

For testnet, the default server url is `https://smash.cardano-testnet.iohkdev.io`. For mainnet, it's something like `https://smash.cardano.iohk.io`, but I didn't have time to check, because mainnet was too outdated in my computer. 

You can indeed use this URL for testing a custom server, but you need to check the network, because if it's testnet, it will lead to a different behaviour in the app, as it's the default server, not a custom one. In this case, you need to use another URL, like the mainnet one.

![Screen Shot 2021-09-17 at 17 21 32](https://user-images.githubusercontent.com/1504716/133850492-8a126879-2357-452e-b6df-2fc14afaa011.png)